modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -1,28 +1,73 @@
+
+== On-Premises Deployments
+
+The following characteristics are specific of Mule runtime engine instances, they only apply for on-premises deployments where you have complete control of the Mule instance.
+
+=== Running Multiple Applications in one Mule Instance
+
+One instance of Mule runtime engine can run multiple applications, enabling you to include the same namespaces within different applications that neither collide nor share information, which allows additional advantages such as:
+
+* You can break down a complex application into several Mule apps with their specific logic, and then deploy the Mule apps in the same Mule instance
+* Clear boundaries for running operations on what a Mule application is
+* Mule runtime engine monitors your Mule applications and reloads configuration changes automatically
+* Applications can depend on different library versions
+* Multiple versions of an application can run within the same Mule instance.
+
+=== Application Package and Deployment
+
+Mule runtime engine unpacks all applications at runtime, removes the original `jar` files inside the `/apps` directory, creates a new folder for each application, and names each folder with the same name of the application file without having the `jar` extension.
+
+To confirm a successful deployment check the following:
+
+* The status of the application in the console is: `DEPLOYED`
+* An unpacked application folder in the `apps` directory of your Mule instance exists +
+For example, for `stockTrader.jar`- `$MULE_HOME/apps/stockTrader`.
+* An anchor file exists for a running app
+For example, `$MULE_HOME/apps/stockTrader-anchor.txt`
+
+If you want to store your applications in a different location, you can store them on Unix-based systems by creating a symlink to your application directory from `$MULE_HOME/apps`.
+
+=== Application and Domain Deployment
+
+Every application is deployed together within a *domain*. By default, your application references the `default` domain, and no further configuration is required. You can see that the domain is deployed with the Mule app in the console:
 
 image::deploy-domain.png[deploy+domain]
 
-If you're deploying multiple applications to the same place, and the applications could share the same resources, then you can create a common *domain* to define configurations that can be referenced by multiple projects. This allows you to expose different services in different projects through the same HTTP host and port without causing any conflicts.
+If you're deploying multiple applications to the same Mule instance, and the applications must share the same resources, then you can create a common *domain* to define configurations that can be referenced by multiple projects. This practice enables you to expose different services in different projects through the same HTTP host and port without causing any conflicts.
+
+=== Application Hot Deployment
```

Comment:
The content of this section is explained with more detail in on-premises-deployment.adoc

Reply:
Additional info is added in PR https://github.com/mulesoft/docs-mule-runtime/pull/1044